src/inlines.c

```diff
@@ -57,6 +57,20 @@ typedef struct subject{
   bool scanned_for_backticks;
 } subject;
 
+// Extensions may populate this.
+static int8_t SKIP_CHARS[256] = {
```

Comment:
You don't need 256 zeroes here. `static int8_t SKIP_CHARS[256];` should do the trick, as static arrays are always zero-initialized.